app/controllers/tasks_controller.rb

```diff
@@ -1,4 +1,57 @@
 class TasksController < ApplicationController
+  before_action :get_list
+  before_action :get_task, only: [:show, :edit, :update, :destroy]
+
   def index
+    @tasks = @list.tasks
+  end
+
+  def new
+    @task = @list.tasks.build
+  end
+
+  def create
+    @task = @list.tasks.build(task_params)
+
+    if @task.save
+      redirect_to @task.list, notice: "Task `#{@task.title}` was successfully created."
+    else
+      render :new, status: :unprocessable_entity
```

Comment:
should we move the whole `create` action content into `new` as it will render the `new.html.erb` anyway?

Reply:
Is that something we should do?
I don't understand the logic in that...

Reply:
my bad, never mind :)